runtime-modules/storage/src/data_object_storage_registry.rs

```diff
@@ -25,10 +25,14 @@
 use codec::{Codec, Decode, Encode};
 use rstd::prelude::*;
 use sr_primitives::traits::{MaybeSerialize, Member, SimpleArithmetic};
-use srml_support::{decl_error, decl_event, decl_module, decl_storage, ensure, Parameter};
+use srml_support::{
+    decl_error, decl_event, decl_module, decl_storage, ensure, print, Parameter, StorageValue,
```

Comment:
adding `StorageValue` seems un-necessary. Did clippy complain?

Reply:
Rust thing: a trait declaration should be in scope.

Reply:
correct, but if I'm not mistaken the `decl_storage!` macro now brings the necessary ones into scope.. like StorageMap, StorageValue etc.. anyway its not a big deal if it stays.

Reply:
Hmm. You are right. It compiles without it. There were errors previously.